Add `@libsql/serverless` package

We already support this via the `@libsql/client/web` import, but people are not associating it with AWS Lambda or Vercel and instead tripping over `@libsql/client` import pulling the native library. Let's do a `@libsql/serverless` package that people can import instead.

Also facing the same issue, 


```
web:deploy: > Using @sveltejs/adapter-cloudflare-workers
web:deploy: ✘ [ERROR] Could not resolve "child_process"
web:deploy:
web:deploy:     ../../node_modules/detect-libc/lib/detect-libc.js:6:29:
web:deploy:       6 │ const childProcess = require('child_process');
web:deploy:         ╵                              ~~~~~~~~~~~~~~~
web:deploy:
web:deploy:   The package "child_process" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
web:deploy:
web:deploy: ✘ [ERROR] Could not resolve "fs"
web:deploy:
web:deploy:     ../../node_modules/detect-libc/lib/filesystem.js:6:19:
web:deploy:       6 │ const fs = require('fs');
web:deploy:         ╵                    ~~~~
web:deploy:
web:deploy:   The package "fs" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
web:deploy:
web:deploy: ✘ [ERROR] Could not resolve "path"
web:deploy:
web:deploy:     ../../node_modules/@neon-rs/load/dist/index.js:27:34:
web:deploy:       27 │ const path = __importStar(require("path"));
web:deploy:          ╵                                   ~~~~~~
web:deploy:
web:deploy:   The package "path" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
web:deploy:
web:deploy: ✘ [ERROR] Could not resolve "fs"
web:deploy:
web:deploy:     ../../node_modules/@neon-rs/load/dist/index.js:28:32:
web:deploy:       28 │ const fs = __importStar(require("fs"));
web:deploy:          ╵                                 ~~~~
web:deploy:
web:deploy:   The package "fs" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
web:deploy:
```

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
                 "edge-light": "./lib-esm/web.js",
                 "netlify": "./lib-esm/web.js",
                 "node": "./lib-esm/node.js",
+                "browser": "./lib-esm/web.js",
                 "default": "./lib-esm/node.js"
             },
             "require": "./lib-cjs/node.js"

